metrics/reporter_interfaces.go

```diff
@@ -25,4 +25,5 @@ type Reporter interface {
 	ReportCount(metric string, tags map[string]string, count float64) error
 	ReportSummary(metric string, tags map[string]string, value float64) error
 	ReportGauge(metric string, tags map[string]string, value float64) error
+	ReportEvent(title string, text string) error
```

Comment:
The other methods also accept `tags`. Do events support tags? If so, we may add them to this method as well.

Reply:
As far as I looked events do not support tags